src/OpenFeature.Extensions.Hosting/OpenFeatureBuilder.cs

```diff
@@ -0,0 +1,9 @@
+using Microsoft.Extensions.DependencyInjection;
+
+namespace OpenFeature;
+
+/// <summary>
+/// Describes a <see cref="OpenFeatureBuilder"/> backed by an <see cref="IServiceCollection"/>.
+/// </summary>
+/// <param name="ServiceCollection"><see cref="IServiceCollection"/></param>
+public sealed record OpenFeatureBuilder(IServiceCollection ServiceCollection);
```

Comment:
I would like to suggest renaming the `ServiceCollection` parameter to `Services`. The reasoning for this suggestion is to align with established .NET conventions where simpler, more generic names are often preferred for widely applicable interfaces or types. This naming convention is consistent with other .NET foundational libraries.

For instance: 

- in Microsoft.Extensions.Logging, the builder uses ILoggingBuilder with methods that refer to services in a general sense ([see the implementation here](https://github.com/dotnet/runtime/blob/main/src/libraries/Microsoft.Extensions.Logging.Abstractions/src/ILoggingBuilder.cs)). 
- OpenTelemetry uses IOpenTelemetryBuilder, emphasizing a streamlined naming convention ([see implementation here](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Api.ProviderBuilderExtensions/IOpenTelemetryBuilder.cs#L12)).

Reply:
Renamed here: [Renamed per suggestion.](https://github.com/open-feature/dotnet-sdk/pull/181/commits/cf1f05fd6d0c3464b9c1a9c2e7eda1b4a658a90b)